packages/flutter/lib/src/material/choice_chip.dart

```diff
@@ -221,15 +221,15 @@ class _FilterChipDefaultsM3 extends ChipThemeData {
     : Theme.of(context).colorScheme.onSurface.withOpacity(0.12);
 
   @override
-  Color? get checkmarkColor => Theme.of(context).colorScheme.onSecondaryContainer;
```

Comment:
please fix

Reply:
I was thinking to add some changes for both Chip and Checkbox in a separate PR, but it makes sense to fix this here. Fixing!